src/theory/strings/regexp_operation.h

```diff
@@ -41,7 +41,9 @@ namespace strings {
  */
 enum RegExpConstType
 {
-  // the regular expression doesn't contain variables or re.allchar or re.range
+  // the regular expression doesn't contain variables or re.complement or
+  // re.allchar or re.range. Notice that re.complement is grouped in this
+  // case since it can be seen as indirectly defined in terms of re.allchar.
```

Comment:
If we keep `re.complement` together with `re.allchar` and `re.range` then the last sentence should be moved to the comment for `RE_C_CONSTANT`.

Reply:
This comment still needs to be updated.